README.md

```diff
@@ -13,10 +13,18 @@ A docker setup for npmo, split into three main components:
 Install the [Docker Toolbox][docker-toolbox], create a machine and run:
 
 ```
-$ FRONT_DOOR_HOST=$(docker-machine ip $DOCKER_MACHINE_NAME) npm run up
+$ npm run configure
 ```
 
-Then you should be able to login:
+This will download your license, and create a docker-compose.yml file. Followed by:
+
+```
+$ npm run up
```

Comment:
Since the file is generated, should you add a section on how to customize? The problem here is that it is hard to maintain different layers of the system to be scaled in different ways... The only thing I will concentrate is to separate the main layers:

* web: The web UI
* search: The search capability
* registry: The Registry itself
* storage: The libraries data

Those could be initialized in different locations by using different docker-compose.yml files. Users would be able to scale them in the same machine, and adding some magic with Consul or others, add discovery and configuration. That would allow us distribute the entire system.